### DISCUSSION TITLE
Update New Relesase

### DISCUSSION THREAD

**Discussion by oksuzgil** [mynaparrot/plugNmeet-server#323] [General]
Hello, new features are not coming to the server, which is the organization, in the update process. Could the new table creaters have not been added?

"msg": "Error 1146 (42S02): Table 'plugnmeet.pnm_room_analytics' doesn't exist"

It would be reasonable to add a separate script for the update operations and the db operations to be upgraded-deleted between the two versions.

**Comment by jibon57** [comment]
Please check release note carefully: https://github.com/mynaparrot/plugNmeet-server/releases/tag/v1.5.0

**Reply by oksuzgil** [reply]
Hi Jibon, thanks for patiently replying. I missed it, got it done. But I would say consider automating cross-version db operations. Although automating db operations is risky, I think it can be done with rollback.
Good luck.
